config/environments/development.rb

```diff
@@ -66,6 +66,4 @@
   # config.action_cable.disable_request_forgery_protection = true
 
   config.zendesk = nil
-
-  config.ckan_redirection_url = ENV["CKAN_REDIRECTION_URL"]
```

Comment:
Nitpick: ideally this (and production.rb) would be a separate commit, as everything else is just a rename.

Reply:
Yeah, fair cop. I'm not going to split this commit, but I'll try to be more granular next time.